### DISCUSSION TITLE
"The authenticity of host 'github.com (140.82.121.3)' can't be established."

### DISCUSSION THREAD

**Discussion by YW5uaWth** [sociomantic-tsunami/git-hub#316] [Q&A] [answered]
Good day!

I'm trying to put up a blog using a Hugo theme on CodeSandbox.io. I can't use `git clone` on CodeSandbox so I opted to install the git-hub package nix package which can be used on NixOS-native CodeSandbox. I've managed to have `Saved git config hub.username` and `Saved git config hub.oauthtoken` show up after inputting the Github token. However, while I'm entering the Github token (`Command + V`), the asterisks to indicate the inputted token don't show up (I'm letting you know just in case it's important, like if the token wasn't actually received by the prompt).

After inputting `git hub clone -t annikorp/hugo-winston-theme`, this is what happened:

```
Cloning git@github.com:zerostaticthemes/hugo-winston-theme.git to hugo-winston-theme
The authenticity of host 'github.com (140.82.121.3)' can't be established.
ECDSA key fingerprint is SHA256:p2QAMXNIC1TJYWeIOttrVc98/R1BUFWu3/LiyKgUfQM.
Are you sure you want to continue connecting (yes/no/[fingerprint])? yes
Error: git clone --quiet --origin upstream -- git@github.com:zerostaticthemes/hugo-winston-theme.git hugo-winston-theme failed (return code: 128)
Warning: Permanently added 'github.com,140.82.121.3' (ECDSA) to the list of known hosts.
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```
It seems that the token wasn't actually received. Is there any other way I can input it?

**Comment by llucax** [comment, marked as answer]
Hi @annikorp, unfortunately `git-hub` is not an alternative to `git`, but it works on top of it, so when you do `git hub clone` you are doing a `git clone` underneath, so if `git clone` doesn't work, neither will `git hub clone`. Feel free to re-open the discussion if you feel this answer is wrong.

**Reply by YW5uaWth** [reply]
I see! Thank you for your response. I really appreciate it :)
